conductr_cli/resolvers/uri_resolver.py

```diff
@@ -13,13 +13,19 @@
 import urllib
 
 
+UNSUPPORTED_SCHEMES = ['s3']
```

Comment:
Let's remove this in favour of having a resolver be asked whether it supports a specific scheme or not. This way, if an explicit scheme is provided by the user then we can also skip resolvers that we know are not able to handle the scheme e.g. there is only one resolver that can handle `s3:`.